doc/datatable.md

```diff
@@ -1,27 +1,36 @@
 # ASAB WebUI DataTable component
 
 `DataTable` is react reusable component.
-It is used to display common data like numbers and strings alongside with JSON objects.
+It is used to display common data like numbers and strings. 
+
+Also `DataTable` supports `advmode`. When `advmode` is enabled, `DataTable` displays extra column which is json for data passed in as `data` prop. More info about `advmode` can be found in `doc/advmode.md`.
+
+Demo app's path: `asab-webui/demo/src/modules/home/containers/TableContainer.js`.
 
 # Implementation
 
+DataTable can be mounted in its own container as it is implemented in demo app or as a widget in another app's container. In first case you must create container for `DataTable` and connect it to Home Module of your application by adding it's route and navigation item. Example may be found in `demo/src/modules/home/index.js`.
```

Comment:
@titkov49 It does not need to be connected to HomeModule of the application, it can be connected to **any** module. Also I would put there an example how the DataTable component is implemented into the project, eg.:

```
import HomeContainer from './containers/HomeContainer'
import Module from 'asab-webui/abc/Module';
import TableContainer from './containers/TableContainer';


export default class HomeModule extends Module {
	constructor(app, name){
		super(app, "HomeModule");
		app.Router.addRoute({ path: '/', exact: true, name: 'Home', component: HomeContainer });
		app.Router.addRoute({ path: '/Table', exact: true, name: 'Table', component: TableContainer });
		app.Navigation.addItem({
			name: 'Home',
			url: '/',
			icon: 'cil-home',
		})
		app.Navigation.addItem({
			name: 'Table',
			url: '/Table',
			icon: 'cil-chart',
		})
	}
}
```